### PR TITLE
updates to map

### PR DESCRIPTION
**Pull request by jread-usgs**

### CODE REVIEW

**Comment by jread-usgs** [comment]
@aappling-usgs want to take a look at this one?

File: inst/shiny/map/server.R [hunk 1]
```diff
@@ -1,46 +1,52 @@
...
     leaflet::leaflet() %>%
       leaflet::addProviderTiles("CartoDB.Positron",
                        options = leaflet::providerTileOptions(noWrap = TRUE)
       ) %>%
-      leaflet::addCircleMarkers(data = data, radius = 10, color=~pal(data[[input$variable]]), 
+      leaflet::addCircleMarkers(data = data, radius = 10, color=~ramp(data[[input$variable]]),
                        popup = ~paste0(long_name, '<br/>', 
-                                       sprintf("<a href='%s' target=_blank>%s</a>",sb_url,site_name), '<br/> value: ',data[[input$variable]]))
+                                      sprintf("<a href='https://www.sciencebase.gov/catalog/folder/%s' target=_blank>%s</a>",sciencebase_id,site_name), '<br/> ',
+                                      tail(strsplit(input$variable,'[.]')[[1]],1),': ',data[[input$variable]],' (',units[[input$variable]],')'))
```
Comment (aappling-usgs) [inline inst/shiny/map/server.R]: the units may not actually need parentheses here - consider "15 mins" vs "15 (mins)"

**Comment by aappling-usgs** [comment]
the color scale could use a legend, if that's not too hard. other than that, i think this map is basically ready to go. looks great.

**Comment by aappling-usgs** [comment]
has travis been failing this whole time? the 403 error is familiar, but i forget what the issue is - github hiccup? adjustments to .travis.yml - maybe need `sudo: required` and maybe also JAGS installation?

**Comment by aappling-usgs** [comment]
the code in the mda.streams travis file might help: https://github.com/USGS-R/mda.streams/blob/master/.travis.yml

File: inst/shiny/map/server.R (hunk 1 above)
Comment (jread-usgs) [inline inst/shiny/map/server.R]: doing.

**Pull request merged**